code/frameworks/sveltekit/package.json

```diff
@@ -65,7 +65,7 @@
   },
   "peerDependencies": {
     "svelte": "^4.0.0 || ^5.0.0-next.16",
-    "vite": "^4.0.0"
+    "vite": "^4.0.0 | ^5.0.0"
```

Comment:
I think there's a typo here, should be two pipes.
You might also need to run `yarn install` in `code` dir to update the lock file, can't remember if it's necessary for peer deps or not 🙂

Reply:
Did it, thanks...pesky pipes i always get confused